self-host/customize-deployment/service-accounts.mdx

```diff
@@ -0,0 +1,300 @@
+---
+title: "Service accounts"
+description: "Configure machine-to-machine authentication for automated API access"
+---
+
+Service accounts provide machine-to-machine authentication for automated access to the Lightdash API. Unlike personal access tokens which are tied to individual users, service accounts are organization-level entities designed for applications, CI/CD pipelines, and automated workflows.
+
+<Info>
+  Service accounts require an **Enterprise License Key**.
+</Info>
+
+## Overview
+
+Service accounts allow you to:
+
+- Authenticate API requests without using personal user credentials
+- Grant specific permission scopes (read, edit, or admin)
+- Set expiration dates for tokens
+- Track token usage and last access time
+- Rotate credentials without affecting user accounts
```

Comment:
we don't support "rotation" at least on the UI 

<img width="1033" height="281" alt="Image" src="https://github.com/user-attachments/assets/cac82bde-df74-47fd-b552-611332dd288b" />